feature/pack/src/main/resources/modules/system/layers/fuse/org/apache/camel/component/stomp/main/module.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module xmlns="urn:jboss:module:1.1" name="org.apache.camel.component.stomp">
+  <resources>
+    <artifact name="${org.apache.camel:camel-stomp}" />
+  </resources>
+  <dependencies>
+    <module name="org.slf4j" />
+    <module name="org.fusesource.stompjms.stompjms-client" export="true" />
```

Comment:
If you export this module and don't share it with other modules, it might as well be aggregated into the component module i.e. be a resource rather than a dependency
(cosmetic)

Reply:
Thanks, fixed in ad19890